src/main/java/com/kodilla/ecommercee/domain/dto/GroupDto.java

```diff
@@ -1,4 +1,27 @@
 package com.kodilla.ecommercee.domain.dto;
 
+import com.kodilla.ecommercee.domain.Product;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+

```

Comment:
It must be added otherwise builder pattern wont work
```suggestion
@NoArgsConstructor
@AllArgsConstructor
```